packages/video_player/video_player_windows/example/.gitignore

```diff
@@ -0,0 +1 @@
+lib/generated_plugin_registrant.dart
```

Comment:
This should never exist in this package AFAIK; was this something you actually saw?

Reply:
Nope, just a copy/paste mistake. I can remove it.